snmp/datadog_checks/snmp/data/default_profiles/dell-os10.yaml

```
@@ -0,0 +1,231 @@
+extends:
+  - _base.yaml
+  - _generic-host-resources-base.yaml
+  - _generic-ucd.yaml
+#  - dell.yaml  # TODO: check hierarchy
+#  - if32-mib.yaml  # TODO: later (NDM-1933)
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "dell"
+
+sysobjectid:
+  - 1.3.6.1.4.1.674.11000.5000.100.2.*      # OS10 family
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.1    # s6000on
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.9    # s4128Ton
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.20   # s5232Fon
+  - 1.3.6.1.4.1.674.11000.5000.100.2.1.21   # s5248Fon
+
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    table:
+      OID: 1.3.6.1.2.1.25.3.3
+      name: hrProcessorTable
+    symbols:
+      - OID: 1.3.6.1.2.1.25.3.3.1.2
+        name: cpu.usage
+    metric_tags:
+      - tag: hr_processor_frw_id
+        column:
+          OID: 1.3.6.1.2.1.25.3.3.1.1
+          name: hrProcessorFrwID
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.used
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1
+  - MIB: DELLEMC-OS10-CHASSIS-MIB
+    symbol:
+      name: os10ChassisTemp
+      OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.3.1.11
+  - MIB: DELLEMC-OS10-CHASSIS-MIB
+    table:
+      OID: 1.3.6.1.4.1.674.11000.5000.100.4.1.1.4
+      name: os10CardTable
```

```suggestion
      name: dell.os10CardTable
```

Shall we prefix all metrics from `DELLEMC-OS10-CHASSIS-MIB` with `dell.` ?

`os10` alone seems not meaningful enough.